subprojects/antlr/build.gradle.kts

```diff
@@ -13,7 +13,7 @@ dependencies {
     implementation(project(":core"))
     implementation(project(":language-jvm"))
     implementation(project(":plugins"))
-    implementation(project(":plugins-java"))
+    implementation(project(":plugins-java-base"))
```

Comment:
`AntlrPlugin` directly references `JavaLibraryPlugin.class`. Shouldn't we also have a dependency on `plugins-java`?

This might apply elsewhere

Reply:
Looks like `JavaLibraryPlugin` is still in `:plugins` but should be moved to `:plugins-java`

Reply:
Yep.  We'll need this eventually, but not yet.

Reply:
Moving those other java plugins will be a separate PR, this one is big enough.  When I do it I'll try removing all references to `plugins` from JVM platform projects and seeing which of them need to be changed.